packages/docs/src/styles/pages/layout.scss

```diff
@@ -21,33 +21,38 @@ body {
   flex: 1 0 auto;
 }
 
+.page-content__content {
+  > h2 {
+    @extend .ds-h2;
```

Comment:
Trying to phase out these `ds-hx` classes in favor of `ds-text-heading--5xl` `ds-text-heading--4xl` etc...   
Could we update to use those instead?

Reply:
Good call, didn't know about that. :+1: